docs/docs/architecture/secrets.md

```diff
@@ -22,3 +22,16 @@ It needs to be provided with the secret seed, from which it can derive the signi
 This procedure is called recovery and is initiated by the workload owner.
 The CLI decrypts the secret seed using the private seed share owner key, verifies the Coordinator and sends the seed through the `Recover` method.
 The Coordinator recovers its key material and verifies the manifest history signature.
+
+## Workload Secrets
+
+The Coordinator provides each workload a secret seed during attestation. This secret can be used by the workload to derive additional secrets for example to
+encrypt persistent data. Like the workload certificates it's mounted in the shared Kubernetes volume `contrast-secrets` in the path `<mountpoint>/secrets/workload-secret`.
+
+:::warning
+
+The data owner shouldn't encrypt data with secrets derived from the workload secret if they want to keep the data private against the workload owner.
```

Comment:
How should the data owner encrypt anything? They aren't running any code, they are just using a service.

Reply:
Fair point. I wrote it with the implicit assumption in mind that the container provider is also the data owner and the workload owner is a third party who only maintains the application by the customer. This is not always the case. 
I'll try to re-word it.